specs-go/config.go

```diff
@@ -320,6 +320,20 @@ type LinuxNetwork struct {
 	Priorities []LinuxInterfacePriority `json:"priorities,omitempty"`
 }
 
+// LinuxRdma for Linux cgroup 'rdma' resource management (Linux 4.11)
+type LinuxRdma struct {
+	// Limits are a set of key value pairs that define RDMA resource limits,
+	// where the key is device name and value is resource limits.
+	Limits map[string]LinuxRdmaLimit `json:"limits,omitempty"`
```

Comment:
This is whatI think you want for `rdma`; no need for a `Limits`.  Just use:

```go
type LinuxRdma map[string]LinuxRdmaLimit
```

Or skip `LinuxRdma entirely and just inline `map[string]LinuxRdmaLimit` in `LinuxResources`.

Reply:
> Or skip LinuxRdma entirely and just inlinemap[string]LinuxRdmaLimitinLinuxResources`.
Simplification looks fine to me. Sending updates.